### PR TITLE
Auth redirect

### PR DESCRIPTION
**Pull request by techtheo**
new

### CODE REVIEW

File: src/plugins/vuetify.js
Comment (samthatcode) [inline src/plugins/vuetify.js:1]: There was no need to Import Vuetify components as they are already available when you install the library. All u needed to do was import the component and directives like this:

// Vuetify
import 'vuetify/styles'
import { createVuetify } from 'vuetify';
import * as components from 'vuetify/components';
import * as directives from 'vuetify/directives';

and register it like this:

const vuetify = createVuetify({
    components,
    directives,
});


app.use(vuetify);

File: src/router/index.js
Comment (samthatcode) [inline src/router/index.js:1]: You should lazy load components that are routed like this:  {
      path: '/about',
      name: 'about',
      component: () => import('../views/AboutView.vue')
    }
    
    It enhances the performance of the web page and the components are only loaded when they are routed to, It's code splitting and it means that they won't load when the app first mounts until they are visited.

File: src/plugins/vuetify.js
Comment (samthatcode) [inline src/plugins/vuetify.js:1]: There was no need to Import Vuetify components as they are already available when you install the library. All u needed to do was import the component and directives like this:

// Vuetify
import 'vuetify/styles'
import { createVuetify } from 'vuetify';
import * as components from 'vuetify/components';
import * as directives from 'vuetify/directives';

and register it like this:

const vuetify = createVuetify({
    components,
    directives,
});


app.use(vuetify);

File: src/components/UserProfile.vue
Comment (samthatcode) [inline src/components/UserProfile.vue:1]: I was able to update my username and email without signing up with those credentials. Fix that bug: You have to do a check. also i should not even be able to update that credentials if im not a user or if i have not signed up for an account

File: src/views/BookEvent.vue
Comment (samthatcode) [inline src/views/BookEvent.vue:1]: I was able to click on the Proceed to payment button and it was successful without booking an event. Also, the Venue filed that is supposed to be a drop down, venues available didn't get populated. Fix that.

**Review by samthatcode — CHANGES_REQUESTED** [review]
Overall, Your code is buggy and needs to be fixed, Please follow convention and best practice. Remove codes that you don't use to ensure clean code and style. Comment out your console.log statement after you have tested things out.

**Comment by techtheo** [comment]
Added  booking features

**Review by techtheo — COMMENTED** [review]
added new features

**Comment by techtheo** [comment]
fixed paystack payment gateway

**Comment by techtheo** [comment]
The Proceed to Payment button now working perfectly well on the ViewBookEvent page

**Comment by techtheo** [comment]
Fully completed features for user and admin

**Pull request merged**